contracts/AirbroCampaignFactory.sol

```diff
@@ -83,15 +87,21 @@ contract AirbroCampaignFactory is AirdropAdmin, AirdropBeta {
 
         NewERC1155DropCampaign airdropContract = NewERC1155DropCampaign(Clones.clone(erc1155DropCampaign));
         airdropContract.initialize(
+            name,
+            symbol,
             uri,
-            address(this) // airBroFactory contract address -> used for getting back admin contract address in airdrop contracts
+            address(this) // airBroFactory contract address
```

Comment:
maybe use msg.sender in airdrop contract's initializer, i think that it is a bit cheaper gas-wise